### PR TITLE
[Playground] Add an option for Datastore namespace value for ci_cd.py script

### LINKED ISSUE #24815: [Task]: Adapt Playground CI/CD scripts to support GCP Datastore multitenancy
### What needs to happen?

Add an input variable for CI/CD scripts to support custom namespaces for Datastore client. 

### Issue Priority

Priority: 3 (nice-to-have improvement)

### Issue Components

- [ ] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam examples
- [X] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

### PR DESCRIPTION
**Pull request by TSultanov**
Add an input variable for CI/CD scripts to support custom namespaces for Datastore client

fixes #24815

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: playground/infrastructure/ci_cd.py [hunk 1]
```diff
@@ -43,6 +43,12 @@
     "beam examples/tests/katas and their outputs on the GCD",
     choices=[Config.CI_STEP_NAME, Config.CD_STEP_NAME],
 )
+parser.add_argument(
```
Comment (olehborysevych) [inline playground/infrastructure/ci_cd.py:46]: @TSultanov  could you please update playground/README.md with info regarding new script argument

File: playground/infrastructure/config.py [hunk 2]
```diff
@@ -113,9 +114,9 @@ class DatastoreProps:
 class RepoProps:
     REPO_DATASETS_PATH = "../backend/datasets"
 
+
```
Comment (olehborysevych) [inline playground/infrastructure/config.py:117]: please revert extra line

File: playground/infrastructure/datastore_client.py [hunk 3]
```diff
@@ -44,10 +44,10 @@ class DatastoreClient:
 
     _datastore_client: datastore.Client
 
-    def __init__(self):
+    def __init__(self, namespace: str):
         self._check_envs()
         self._datastore_client = datastore.Client(
-            namespace=DatastoreProps.NAMESPACE, project=Config.GOOGLE_CLOUD_PROJECT
+            namespace=namespace, project=Config.GOOGLE_CLOUD_PROJECT
```
Comment (olehborysevych) [inline playground/infrastructure/datastore_client.py:50]: I'm not comfortable that we have two related properties of the datastore client but they are obtained from different sources. Maybe it's not too late to reconsider this approach and have both as env variables or arguments? @TSultanov

**Review by olehborysevych — CHANGES_REQUESTED** [review]

File: playground/infrastructure/datastore_client.py (hunk 3 above)
Comment (TSultanov) [reply playground/infrastructure/datastore_client.py]: Moved this configuration option from environment variable to the `ci_cd.py` script arguments and updated the ci/cd configs which use this script

**Review by damondouglas — APPROVED** [review]
@pabloem LGTM

**Comment by pabloem** [comment]
lgtm thanks y'all!

**Pull request merged**